src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -3,18 +3,139 @@
 import React from "react";
 import { Meta, Story } from "@storybook/react";
 import { Tooltip as TooltipComponent, TooltipProps } from "./Tooltip";
+import { BrightHeaderVariants } from "./partials/BrightHeader";
+import IconIcons from "@elements/Icon/Generated/IconIcons";
 
 export default {
     title: "Components/Tooltip",
     component: TooltipComponent,
     args: {
-        tooltip: "Tooltip Content (Reactnode)",
+        tooltip: "Your text here",
+        heading: "",
+        buttons: [],
+    },
+    argTypes: {
+        brightHeader: {
+            options: [
+                null,
+                BrightHeaderVariants.Information,
+                BrightHeaderVariants.Warning,
+                BrightHeaderVariants.Tip,
+                BrightHeaderVariants.Note,
+            ],
```

Comment:
You could do the same mapping as the following `argType` so we have better info than `null` when selecting the header type